sycl/tools/abi_check.py

```diff
@@ -0,0 +1,106 @@
+import argparse
```

Comment:
This module needs comments - purpose, in which cases when the check is considered passed, in which - failed